www/templates/jsontestresults.html

```diff
@@ -27,7 +27,7 @@
                 <div class="clearfix"></div>
                 {% for artifact in suite.artifacts %}
                     <div class="artifact">
-                        <a target="_blank" href="{{ join(path_to_artifacts, artifact) }}">{{ basename(artifact) }}</a>
+                        <a target="_blank" href="{{ join(path_to_artifacts, artifact).replace("\\","/") }}">{{ basename(artifact.replace("\\","/")) }}</a>
```

Comment:
No, `path_to_artifacts` will never contain '\', the `artifact` can have '\', `urljoin` works the same way as path join, it doesn't replace backslashes to forwardslashes when joins